src/test/java/rest/CountryResourceTest.java

```diff
@@ -81,7 +81,17 @@ public void testWrongGetCountry() throws Exception {
                 .contentType("application/json")
                 .get("/resource/WRONG").then()
                 .assertThat()
-                .statusCode(400) 
+                .statusCode(400)
                 .body("message", equalTo("No country by that name exists."));
```

Comment:
The <code>/resource/{parameter}</code> path leads to another endpoint. Even if you type <code>/resource/countryname/</code> (with the slash at the end) - you will still reach another endpoint. 
The correct path for testing a bad case using the alpha2/countryname endpoint could look like this: 
<code>/resource/countryname/BadAlpha2Code</code>. 
This should lead to a code 400 with the message: <code>"No country with given alpha2 code found"<code>

Reply:
That test is testing this endpoint :) 
![image](https://user-images.githubusercontent.com/11192971/69255388-042a7200-0bb8-11ea-803a-6fabf8f200f8.png)

But yeah, I even confused myself while coding it. Spent like 20 minutes on a wrong endpoint wondering why I got 404

Reply:
Oh snap, my bad! I just assumed it was for the other endpoint because the title says "Tests for Asger Methods" x) Sorry :)